lib/cf/broker/manifest.yml

```diff
@@ -7,11 +7,11 @@ applications:
   disk_quota: 512M
   env:
     CONF: default
-    DEBUG: e-abacus-*
+    DEBUG: metering-*,e-metering-*
```

Comment:
I think we need to have `e-abacus-*` as well to support error debugs for used modules.